Tests/Unit/NetworkServer/ConfigurationTest.cs

```diff
@@ -39,5 +39,31 @@ public void Should_Setup_Allowed_Dev_Addresses_Correctly(string inputAllowedDevA
             TheoryDataFactory.From(("0228B1B1;", new[] { new DevAddr(0x0228b1b1) }),
                                    ("0228B1B1;0228B1B2", new DevAddr[] { new DevAddr(0x0228b1b1), new DevAddr(0x0228b1b2) }),
                                    ("ads;0228B1B2;", new DevAddr[] { new DevAddr(0x0228b1b2) }));
+        [Theory]
+        [InlineData("500")]
+        [InlineData("x")]
+        public void ProcessingDelayIsConfigurable(string processing_delay)
+        {
+            var envVariables = new[] { ("PROCESSING_DELAY_IN_MS", processing_delay) };
+
+            try
+            {
+                foreach (var (key, value) in envVariables)
+                    Environment.SetEnvironmentVariable(key, value);
```

Comment:
Note that this can interfere with other tests running in parallel (xUnit runs unit tests from different test classes in parallel per default) - consider using `IConfiguration` to isolate this and be independent of side effects.

Reply:
We can't use IConfiguration as the NetworkServerConfiguration is directly parsing environment variables. Let's keep as @andrewdoing did

Reply:
Linking the story with the required change: #1010